### DISCUSSION TITLE
Resize markdown table in reveal js

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#5156] [Q&A] [answered]
I have a markdown table that, when rendered, is suprisingly large. I'd like to resize it so that it's not so big and preferably make it a striped table. 

```
---
title: "geohash precision table"
format: revealjs
---

## 

| Index Precision | Bounding Box         |
|-----------------|----------------------|
| 1               | ≤ 5,000km X 5,000 Km |
| 2               | ≤ 1,250km X 625km    |
| 3               | ≤ 156km X 156km      |
| 4               | ≤ 39.1km X 19.5km    |
| 5               | ≤ 4.89km X 4.89km    |
| 6               | ≤ 1.22km X 0.61km    |
| 7               | ≤ 153m X 153m        |
| 8               | ≤ 38.2m X 19.1m      |
| 9               | ≤ 4.77m X 4.77m      |
| 10              | ≤ 1.19m X 0.596m     |
| 11              | ≤ 149mm X 149mm      |
| 12              | ≤ 37.2mm X 18.6mm    |

```

Here is what it looks like when rendered. 

<img width="1511" alt="image" src="https://user-images.githubusercontent.com/13534508/231224743-59ca1774-78c7-49de-af4f-b34399f85dd9.png">

I'd prefer to not have a supplementary css file to go along with the quarto doc.

**Comment by cscheid** [comment, marked as answer]
> I'd prefer to not have a supplementary css file to go along with the quarto doc.

Is your preference about avoiding an actual additional file or an extra style declaration? If it's the former, then you can use 

````
---
title: "geohash precision table"
format: revealjs
include-in-header: 
  text: |
    <style>
      .custom-small table {
        font-size: .5em
      }
    </style>
---

## 

::: custom-small

| Index Precision | Bounding Box         |
|-----------------|----------------------|
| 1               | ≤ 5,000km X 5,000 Km |
| 2               | ≤ 1,250km X 625km    |
| 3               | ≤ 156km X 156km      |
| 4               | ≤ 39.1km X 19.5km    |
| 5               | ≤ 4.89km X 4.89km    |
| 6               | ≤ 1.22km X 0.61km    |
| 7               | ≤ 153m X 153m        |
| 8               | ≤ 38.2m X 19.1m      |
| 9               | ≤ 4.77m X 4.77m      |
| 10              | ≤ 1.19m X 0.596m     |
| 11              | ≤ 149mm X 149mm      |
| 12              | ≤ 37.2mm X 18.6mm    |

:::
````

**Reply by JosiahParry** [reply]
This works well enough! A second file is the largest hesitance followed by CSS. I don't like this tooo much but it works just fine! 

Out of curiosity what is the difference between `::: {.class}` syntax vs `::: class`?

**Reply by cscheid** [reply]
> This works well enough! A second file is the largest hesitance followed by CSS. I don't like this tooo much but it works just fine!

Got it. I'm not sure what else we could do in this case, though!


> Out of curiosity what is the difference between `::: {.class}` syntax vs `::: class`?

There isn't one - the latter is a shortcut for the former.
